bin/yarn

```diff
@@ -1 +1 @@
-../../../root/.cache/hermit/pkg/node-18.9.1/lib/node_modules/corepack/dist/yarn.js
\ No newline at end of file
+../../../root/.cache/hermit/pkg/node-19.8.1/lib/node_modules/corepack/dist/yarn.js
```

Comment:
This `yarn` is broken: https://github.com/cashapp/hermit-packages/issues/253
I found out we can actually get `yarn` from https://www.npmjs.com/package/corepack
So I'm trying this out now separately.